vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -1,33 +1,90 @@
 """Unit tests for vizro.models.Graph."""
 import pytest
+from dash import dcc
+from plotly import graph_objects as go
 from pydantic import ValidationError
 
 import vizro.models as vm
 import vizro.plotly.express as px
+from vizro.managers import data_manager
+from vizro.models._components.graph import create_empty_fig
 
 
-def test_create_graph(standard_px_chart):
-    graph = vm.Graph(figure=standard_px_chart)
+class TestCall:
+    def test_create_graph(self, standard_px_chart):
+        graph = vm.Graph(figure=standard_px_chart)
 
-    assert hasattr(graph, "id")
-    assert graph.type == "graph"
-    assert graph.figure == standard_px_chart._captured_callable
-    assert graph.actions == []
+        assert hasattr(graph, "id")
+        assert graph.type == "graph"
+        assert graph.figure == standard_px_chart._captured_callable
+        assert graph.actions == []
 
+    def test_failed_graph_with_wrong_figure(self, standard_go_chart):
+        with pytest.raises(ValidationError, match="must provide a valid CapturedCallable object"):
+            vm.Graph(
+                figure=standard_go_chart,
+            )
 
-def test_failed_graph_with_wrong_figure(standard_go_chart):
-    with pytest.raises(ValidationError, match="must provide a valid CapturedCallable object"):
-        vm.Graph(
-            figure=standard_go_chart,
+    @pytest.mark.parametrize("title, expected", [(None, 24), ("Test", None)])
```

Comment:
Shall we add a test for the __getitem__ method of the Graph? Then I think we cover everything in the instantiation